honglidong/app/src/main/java/com/hongildong/map/ui/search/location_detail/facility/review/FacilityReviewTab.kt

```
@@ -150,28 +192,46 @@ fun FacilityReviewTab(
+fun FacilityReviewInfo(
+    recommendInfo: ReviewRecommendResponse
+) {
+    Column {
+        Row (
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(vertical = 25.dp),
+            verticalAlignment = Alignment.CenterVertically
+        ) {
+            Text(
+                String.format("%.1f", recommendInfo.avgRating),
```

detekt (1.23.8)

[warning] String.format("%.1f", recommendInfo.avgRating) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)